lib/mix/tasks/phx.gen.html.ex

```diff
@@ -0,0 +1,138 @@
+Code.require_file("../../../installer/lib/phx_new/generator.ex", __DIR__)
```

Comment:
right. I just double-checked something, and it seems compilation gets through if archive was installed (I used: `cd installer && MIX_ENV=prod mix archive.build && mix archive.install`). That's probably why I haven't seen it before.